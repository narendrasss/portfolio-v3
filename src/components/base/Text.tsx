import React from 'react';
import { Text as BaseText, TextProps } from 'rebass';

export const Text: React.FC<TextProps> = props => (
  <BaseText as="p" fontFamily="sans" fontSize={[0, 1]} {...props} />
);

export const Mono: React.FC<TextProps> = props => (
  <BaseText as="p" fontFamily="mono" fontSize={[0, 1]} {...props} />
);
