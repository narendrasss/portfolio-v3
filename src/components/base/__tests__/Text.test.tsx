import React from 'react';
import { render } from 'react-testing-library';
import Text from '../Text';

describe('Text', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<Text />);
    expect(asFragment()).toMatchSnapshot();
  });
});
