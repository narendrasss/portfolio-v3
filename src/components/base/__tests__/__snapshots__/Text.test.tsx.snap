// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders without crashing 1`] = `
<DocumentFragment>
  <p
    class="sc-bdVaJa sc-htpNat ejPUVP"
    font-family="sans"
    font-size="0,1"
  />
</DocumentFragment>
`;
